packages/@webex/plugin-meetings/src/constants.ts

```diff
@@ -449,6 +456,30 @@ export const ERROR_DICTIONARY = {
     MESSAGE: 'Captcha is required.',
     CODE: 8,
   },
+  RECLAIM_HOST_ROLE_NOT_SUPPORTED: {
+    NAME: 'ReclaimHostRoleNotSupported',
+    MESSAGE:
+      'Non converged meetings, PSTN or SIP users in converged meetings are not supported currently.',
```

Comment:
Suggestion: Can we have a heading like we have in RECLAIM_HOSt_ROLE_NOT ALLOWED and RECLAIM_HOST_ROLE_EMPTY_OR_WRONG_KEY. Just for uniformity.

Reply:
Discussed 1:1